posts/2022-11-01-instant-on-beta-jaJP.adoc

```diff
@@ -0,0 +1,1163 @@
+---
+layout: post
+title: Liberty InstantOn startup for cloud native Java applications
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tjwatson
+author_github: https://github.com/tjwatson
+seo-title: Liberty InstantOn startup for cloud native Java applications - OpenLiberty.io
+seo-description: The Open Liberty 22.0.0.11-beta brings you an exciting new feature called InstantOn that provides incredibly fast startup times for Jakarta EE and MicroProfile applications.
+blog_description: "The Open Liberty 22.0.0.11-beta brings you an exciting new feature called InstantOn that provides incredibly fast startup times for Jakarta EE and MicroProfile applications."
+open-graph-image: https://openliberty.io/img/blog/instantonperf.png
```

Comment:
```suggestion
open-graph-image: https://openliberty.io/img/blog/instantonperf.png
blog-available-in-languages: ['en']
```

Adding `blog-available-in-languages` front matter to cross link this post to the English version